analyzers/src/SonarAnalyzer.CSharp/Rules/EqualityOnModulus.cs

```diff
@@ -34,7 +37,15 @@ public sealed class EqualityOnModulus : SonarDiagnosticAnalyzer
         private const string DiagnosticId = "S2197";
         private const string MessageFormat = "The result of this modulus operation may not be {0}.";
 
-        private static readonly CSharpExpressionNumericConverter ExpressionNumericConverter = new CSharpExpressionNumericConverter();
+        private const string CountName = nameof(Enumerable.Count);
+        private const string LongCountName = nameof(Enumerable.LongCount);
+        private const string LengthName = "Length";
```

Comment:
nameof(String.Length)

Reply:
I intentionally set it as `"Length"` because it represents both the `Array.Length` and the `String.Length`.
Although `nameof(String.Length)` would give the same result, it would probably be misleading.

Reply:
Lets just put a comment there please that it stands for both.